src/nidcpower/metadata/functions.py

```diff
@@ -230,11 +237,12 @@
                 },
                 'name': 'customCableCompensationData',
                 'python_api_converter_name': 'convert_to_bytes',
+                'python_type': 'bytes',
                 'size': {
                     'mechanism': 'len',
                     'value': 'customCableCompensationDataSize'
                 },
-                'type': 'ViInt8[]',
+                'type': 'ViAddr[]',
```

Comment:
Array of addresses is wrong. Something didn't get set correctly.

Reply:
This is a continued result of the current state of `address`/`ViAddr` parameters. The same thing can be seen [here](https://github.com/ni/nimi-python/blob/master/src/nidmm/metadata/functions.py#L1336), for example.

Reply:
Ah, I see what you're saying David. If the generated python code is correct despite the metadata being incorrect, that's not something to solve here in this PR.